src/main/resources/conf/base.conf

```diff
@@ -253,6 +253,15 @@ mantis {
         peer-manager-timeout = 5.seconds
       }
 
+      health {
+        # If the client does see a change of the best known block number after this duration
+        # the healthcheck will consider it to be stuck and return an error
```

Comment:
I guess you mean `otherwise the healthcheck will consider it to be stuck and return an error`